src/app/settings/sponsored-families.component.ts

```diff
@@ -28,14 +31,21 @@ export class SponsoredFamiliesComponent implements OnInit {
     private i18nService: I18nService,
     private platformUtilsService: PlatformUtilsService,
     private syncService: SyncService,
-    private organizationService: OrganizationService
+    private organizationService: OrganizationService,
+    private validationService: ValidationService,
+    private stateService: StateService
   ) {}
 
   async ngOnInit() {
     await this.load();
   }
 
   async submit() {
+    if (this.currentUserEmail.toLowerCase() === this.sponsorshipEmail.toLowerCase()) {
+      this.validationService.showError(this.i18nService.t("cannotSponsorSelf"));
```

Comment:
This will do a toast correct? I believe design wanted us to inline error validation for this.